### PR TITLE
[netcore] Use msbuild to download tests

### PR DESCRIPTION
**Pull request by EgorBo**
WIP

### CODE REVIEW

File: eng/helixcorefxtests.proj [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <HelixTargetQueue>''</HelixTargetQueue>
+    <TestAssetBlobFeedUrl>https://dotnetfeed.blob.core.windows.net/dotnet-core</TestAssetBlobFeedUrl>
+    <MicrosoftPrivateCoreFxNETCoreAppVersion>4.6.0-preview8.19353.6</MicrosoftPrivateCoreFxNETCoreAppVersion>
+    <__BuildOS>OSX</__BuildOS>
+    <__BuildArch>x64</__BuildArch>
+    <_TargetGroup>netcoreapp</_TargetGroup>
+  </PropertyGroup>
+
+  <!-- Import Microsoft.DotNet.Build.Tasks.Feed.targets for `ParseBuildManifest` -->
+  <Import Project="/Users/egorb/.nuget/packages/microsoft.dotnet.build.tasks.feed/2.2.0-beta.19356.1/build/Microsoft.DotNet.Build.Tasks.Feed.targets" />
```
Comment (filipnavara) [inline eng/helixcorefxtests.proj]: If you planned to check the CI you may need to update this path ;-)

Comment (EgorBo) [reply eng/helixcorefxtests.proj]: nah this PR is more like a sample -- it's too many helix variables I need to set and I have no idea how

File: eng/helixcorefxtests.proj [hunk 2]
```diff
@@ -0,0 +1,285 @@
+<Project InitialTargets="BuildHelixWorkItems" DefaultTargets="RunInParallelForEachScenario">
```
Comment (marek-safar) [inline eng/helixcorefxtests.proj]: Should this file be added manually?

Comment (EgorBo) [reply eng/helixcorefxtests.proj]: @marek-safar it looks like it was added manually in coreclr: https://github.com/dotnet/coreclr/commits/master/eng/helixcorefxtests.proj

the script correctly downloads the tests-manifest and parses it (`ParseBuildManifest` target) but then - I have no idea what it wants from me in order to download them. It was just a quick attempt

**Comment by akoeplinger** [comment]
Discussed with Egor, this is not worth it at the moment, closing.

**Pull request closed without merging**